charts/merlin/templates/mlflow-deployment.yaml

```diff
@@ -29,9 +26,9 @@ spec:
       labels:
         app: {{ template "mlflow.fullname" . }}
         release: {{ .Release.Name }}
-      {{- if .Values.mlflow.podLabels}}
-        {{ toYaml .Values.mlflow.podLabels | nindent 8 }}
-      {{- end}}
+        {{- if .Values.mlflow.podLabels }}
```

Comment:
This can be verified by adding `mlflow.podLabels` in https://github.com/gojek/merlin/blob/main/charts/merlin/values-e2e.yaml

Reply:
Yeah, that makes sense. Added 👍